scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py

```diff
@@ -146,6 +155,36 @@ def get_cluster(self):
     check_type(cluster, dict, "cluster")
     return cluster
 
+def get_node(self):
```

Comment:
get_node and get_cortx is not required as a seperate function, we can use get_value instead